README.md

```diff
@@ -140,12 +140,20 @@ The pulsar-manager can monitor topics and subscriptions.
 
 3. Access Pulsar manager website.
 
-    Use the account and the password to log in to the Pulsar manager website as below.  
-          
+If you use the container to build the environment, please directly using the the account and the password to log in to the Pulsar manager website as below.
+
    * Account: `pulsar`  
    * Password: `pulsar`  
+   
+if you use the latest code to deploy the environment yourself, please use the following command to initialize the superuser and password
```

Comment:
If you are deploying Pulsar Manager using the latest code, you can create a super-user using the following command.